align/compiler/user_const.py

```diff
@@ -30,14 +33,15 @@ def read_user_const(self, design_name: str):
         Reads user defined constraints and create a dictionary for each hierarchy
   
         """
-        self.block_const = {}
+        self.block_const = constraint.ConstraintDB()
         fp = self.input_dir / (design_name+'.const')
         fp_json = self.input_dir / (design_name+'.const.json')
         if fp_json.is_file():
             logger.info(f"JSON input const file for block {design_name} {fp_json}")
-            f = open(fp_json, "r")
-            self.block_const = json.load(f)
+            self.block_const = constraint.ConstraintDB.parse_file(fp_json)
         elif fp.is_file():
+            raise NotImplementedError('Command-line interface has not been upgraded. Please use json constraints')
```

Comment:
Command-line constraint format was highly experimental. Disabled it altogether. I suggest eliminating all relevant code and the layers.json entry for allowed constraints as well. If we really do want this kind of functionality, please consider using a third-party utility like `pydantic-cli` which should be able to create argparse type subparsers directly from our schema.constraint datamodel.